ProjectCoimbra.UWP/DataAccessLibrary/DataAccess.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed under the MIT License.
+
+namespace DataAccessLibrary
+{
+    using Microsoft.Data.Sqlite;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Windows.Storage;
+
+    /// <summary>
+    /// Data access class for the local User table in SQLite database
+    /// </summary>
+    public static class DataAccess
+    {
+        /// <summary>
+        /// Initializes the database, which is called when application starts
+        /// </summary>
+        public async static void InitializeDatabase()
+        {
+            await ApplicationData.Current.LocalFolder.CreateFileAsync("coimbra.db", CreationCollisionOption.OpenIfExists);
```

Comment:
Please suffix all async calls with `.ConfigureAwait(false)` to more effciently use thread resources. If it doesn't work in this case, please instead add `.ConfigureAwait(true)` to make the intention clear and avoid problems with upgrading to .NET Core.